mifosng-android/src/main/java/com/mifos/mifosxdroid/login/LoginActivity.java

```diff
@@ -158,10 +158,8 @@ public boolean validateUserInputs() {
     public void onLoginSuccessful(User user) {
         Toaster.show(findViewById(android.R.id.content), getString(R.string.toast_welcome) + " "
                 + user.getUsername());
-        // Saving userID
-        PrefManager.setUserId(user.getUserId());
-        // Saving user's token
-        PrefManager.saveToken("Basic " + user.getBase64EncodedAuthenticationKey());
```

Comment:
Do not remove these, Token is being used, whenever we are making any request after login.